### PR TITLE
Add `CODEOWNERS`

### PR DESCRIPTION
**Pull request by hassankhan**
**Check List**
- [X] Tests has been run in packages where changes made if available
- [X] Linter has been run for changed code
- [X] Tests for the changes have been added if not covered yet
- [X] Docs have been added / updated if required

**Description of Changes Made (if issue reference is not provided)**

Added a `CODEOWNERS` file so relevant teams can be notified when PRs require a review. More information about the file can be [found here](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-file-location)

### CODE REVIEW

**Review by keydunov — APPROVED** [review]

**Comment by keydunov** [comment]
👍

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# These owners will be the default owners for everything in the repo. Unless a
```
Comment (rpaik) [inline CODEOWNERS:1]:
```suggestion
# The following people will be the default owners for this repo. Unless a
```

Comment (rpaik) [reply CODEOWNERS:1]: This maybe simpler/clearer?

**Comment by igorlukanin** [comment]
I love the idea to add `CODEOWNERS` so much. Thank you, @hassankhan!

**Review by igorlukanin — APPROVED** [review]

**Comment by hassankhan** [comment]
Thanks for the great feedback, guys! I've reworded it as per @rpaik's suggestion 👍

**Review by rpaik — APPROVED** [review]

**Comment by rpaik** [comment]
Thanks @hassankhan, looks good to me 😄 
@paveltiunov @vasilev-alex are you OK with the changes? I'm happy to merge....

**Review by vasilev-alex — APPROVED** [review]

**Comment by vasilev-alex** [comment]
yep 👍 , merging

**Pull request merged**